Sources/SwiftFormatRules/NoEmptyTrailingClosureParentheses.swift

```diff
@@ -37,7 +39,9 @@ public final class NoEmptyTrailingClosureParentheses: SyntaxFormatRule {
       on: node.calledExpression,
```

Comment:
I wonder if we need to recursively visit `node.calledExpression` as well.

I think this test would catch it:

```swift
foo(bar() { baz })() { blah }
```

`node.calledExpression` should be `foo(bar() { baz })`, so if the parens after `bar` don't get dropped, then we need to call it there too.

Reply:
Good call. Added.